packages/filebrowser/src/listing.ts

```diff
@@ -1833,8 +1833,13 @@ export namespace DirListing {
         // clean up the svg icon annotation, if any
         delete icon.dataset.icon;
       }
+      // add file size to pop up if its available
+      if (model.size !== null && model.size !== undefined) {
+        node.title = model.name + ' - ' + Private.formatFileSize(model.size, 1);
```

Comment:
Thanks for adding this!  I think it looks too sparse without adding more info.  I think we should show the following:

Name: foo.bar
Size: as you have it
Path: path limited to 50 chars with an ellipsis if longer
Created:  Formatted using [Time.formatHuman](https://github.com/jupyterlab/jupyterlab/blob/1aff4190084fd5993d3f7e3ae10b467264df1cd0/packages/coreutils/src/time.ts#L17).
Modified: Same as above

Similar to what Finder shows on MacOS about a file:

<img width="251" alt="image" src="https://user-images.githubusercontent.com/2096628/68352862-c61c5100-00d5-11ea-88ee-3c0ddac7ae9b.png">

Reply:
And just not show the size part if we don't have it.

Reply:
Yeah that looks better. Let me fix it.